web/src/engine/osk/src/keyboard-layout/oskBaseKey.ts

```diff
@@ -197,29 +196,23 @@ export default class OSKBaseKey extends OSKKey {
     this.btn.replaceChild(this.preview, oldPreview);
   }
 
-  public refreshLayout(vkbd: VisualKeyboard) {
-    let key = this.spec as ActiveKey;
-    this.square.style.width = vkbd.layoutWidth.scaledBy(key.proportionalWidth).styleString;
-    this.square.style.marginLeft = vkbd.layoutWidth.scaledBy(key.proportionalPad).styleString;
-    this.btn.style.width = vkbd.usesFixedWidthScaling ? this.square.style.width : '100%';
+  public refreshLayout(layoutParams: KeyLayoutParams) {
+    const keyTextClosure = super.refreshLayout(layoutParams);  // key labels in particular.
```

Comment:
Why 'keyTextClosure'? It's really unclear what that does! Perhaps we should call it `superFinalization` to clarify that it's more work that super.refreshlayout has to do once this function is finished?

Reply:
> Why 'keyTextClosure'?

Because it's a closure to finalize/apply layout changes directly tied to key text as seen in OSKKey.  

> Perhaps we should call it `superFinalization` [...]

If I swapped to `superFinalization` or something else unrelated to the code-contents of the closure... well, I suppose I can always just add a comment about what the closure's actual effects are.